perllib/FixMyStreet/Cobrand/HighwaysEngland.pm

```diff
@@ -88,6 +88,18 @@ sub allow_anonymous_reports { 'button' }
 
 sub admin_user_domain { 'highwaysengland.co.uk' }
 
+sub abuse_reports_only { 1 }
+
+sub contact_name {
```

Comment:
If you look near the top of the file, you'll see a number of functions-that-should-use-UKCouncils-code being passed through, would be good to match those rather than repeat the code?